### PR TITLE
Kube-scheduler ID Localization

### PR DESCRIPTION
**Pull request by ariscahyadi**
Hi,

I did a ID translation for `content/en/docs/concepts/scheduling/kube-scheduler`

Thanks.

### CODE REVIEW

File: content/id/docs/concepts/scheduling/kube-scheduler.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: Penjadwal Kubernetes
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+Dalam Kubernetes, _scheduling_ merujuk dalam hal untuk memastikan
```
Comment (irvifa) [inline content/id/docs/concepts/scheduling/kube-scheduler.md:9]:
```suggestion
Dalam Kubernetes, _scheduling_ ditujukan untuk memastikan 
```

File: content/id/docs/concepts/scheduling/kube-scheduler.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+Dalam Kubernetes, _scheduling_ merujuk dalam hal untuk memastikan 
+{{< glossary_tooltip text="Pods" term_id="pod" >}} mendapatkan
+{{< glossary_tooltip text="Nodes" term_id="node" >}} sehingga
+{{< glossary_tooltip term_id="kubelet" >}} dapat menjalankannya.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Ikhtisar Scheduling {#scheduling}
```
Comment (irvifa) [inline content/id/docs/concepts/scheduling/kube-scheduler.md:18]: Will you use scheduling or penjadwalan here?

File: content/id/docs/concepts/scheduling/kube-scheduler.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+Sebuah penjadwal mengawasi Pod yang baru saja dibuat dan belum ada Node yang 
+dialokasikan untuknya. Untuk setiap Pod yang ditemukan oleh penjadwal, maka
+penjadwal tersebut bertanggung jawab untuk menemukan Node terbaik untuk 
+menjalankan Pod. Penjadwal dapat menetapkan keputusan penempatan ini dengan 
+mempertimbangkan prinsip-prinsip penjadwalan yang dijelaskan di bawah ini.
+
+Jika Anda ingin memahami mengapa Pod ditempatkan pada Node tertentu, atau jika
+Anda berencana untuk mengimplementasikan penjadwal kustom sendiri, halaman ini
+akan membantu Anda belajar tentang penjadwalan.
```
Comment (irvifa) [inline content/id/docs/concepts/scheduling/kube-scheduler.md:28]:
```suggestion
akan membantu kamu belajar tentang penjadwalan.
```

File: content/id/docs/concepts/scheduling/kube-scheduler.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+## kube-scheduler
+
+[kube-scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/)
+adalah penjadwal standar untuk Kubernetes dan dijalankan sebagai bagian dari
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}.
+kube-scheduler dirancang agar jika Anda mau dan perlu, Anda bisa menulis
```
Comment (irvifa) [inline content/id/docs/concepts/scheduling/kube-scheduler.md:35]:
```suggestion
kube-scheduler dirancang agar jika kamu mau dan perlu, kamu bisa menulis 
```

File: content/id/docs/concepts/scheduling/kube-scheduler.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+Anda berencana untuk mengimplementasikan penjadwal kustom sendiri, halaman ini
+akan membantu Anda belajar tentang penjadwalan.
+
+## kube-scheduler
+
+[kube-scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/)
+adalah penjadwal standar untuk Kubernetes dan dijalankan sebagai bagian dari
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}.
+kube-scheduler dirancang agar jika Anda mau dan perlu, Anda bisa menulis 
+komponen penjadwalan Anda sendiri dan menggunakannya.
```
Comment (irvifa) [inline content/id/docs/concepts/scheduling/kube-scheduler.md:36]:
```suggestion
komponen penjadwalan kamu sendiri dan menggunakannya.
```

File: content/id/docs/concepts/scheduling/kube-scheduler.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+Ada dua cara yang didukung untuk mengonfigurasi perilaku penyaringan dan
+penilaian oleh penjadwal:
+
+1. [Aturan Scheduling](/docs/reference/scheduling/policies) yang memungkinkan 
+   Anda untuk mengkonfigurasi _Predicates_ untuk pemfilteran dan _Priorities_ 
+   untuk penilaian.
+1. [Profil Scheduling](/docs/reference/scheduling/profiles) yang memungkinkan
+   Anda mengonfigurasi _Plugin_ yang menerapkan tahapan penjadwalan berbeda, 
+   termasuk: `QueueSort`, `Filter`, `Score`, `Bind`, `Reserve`, `Permit`, dan 
+   lainnya. Anda juga bisa mengonfigurasi kube-scheduler untuk menjalankan
```
Comment (irvifa) [inline content/id/docs/concepts/scheduling/kube-scheduler.md:91]:
```suggestion
   lainnya. Kamu juga bisa mengonfigurasi kube-scheduler untuk menjalankan 
```

File: content/id/docs/concepts/scheduling/kube-scheduler.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+memilih salah satunya secara acak.
+
+Ada dua cara yang didukung untuk mengonfigurasi perilaku penyaringan dan
+penilaian oleh penjadwal:
+
+1. [Aturan Scheduling](/docs/reference/scheduling/policies) yang memungkinkan 
+   Anda untuk mengkonfigurasi _Predicates_ untuk pemfilteran dan _Priorities_ 
+   untuk penilaian.
+1. [Profil Scheduling](/docs/reference/scheduling/profiles) yang memungkinkan
+   Anda mengonfigurasi _Plugin_ yang menerapkan tahapan penjadwalan berbeda,
```
Comment (irvifa) [inline content/id/docs/concepts/scheduling/kube-scheduler.md:89]:
```suggestion
   Kamu mengonfigurasi _Plugin_ yang menerapkan tahapan penjadwalan berbeda, 
```

File: content/id/docs/concepts/scheduling/kube-scheduler.md (hunk 2 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/scheduling/kube-scheduler.md:18]: I think better penjadwalan, I just worried about the hashtag. But I know that I need to change the hashtag also.

**Comment by irvifa** [comment]
Hi @ariscahyadi can you please squash your commits into one?

**Comment by ariscahyadi** [comment]
> Hi @ariscahyadi can you please squash your commits into one?



> Hi @ariscahyadi can you please squash your commits into one?

In my repo? I am not sure if I can do that.

**Comment by ariscahyadi** [comment]
> > Hi @ariscahyadi can you please squash your commits into one?
> 
> > Hi @ariscahyadi can you please squash your commits into one?
> 
> In my repo? I am not sure if I can do that.

It is done! Thanks.

**Comment by girikuncoro** [comment]
this is looking good, thanks for your contribution! 🎉 please don't hesitate to help translating other pages 😄 

/lgtm
/approve

**Pull request merged**